openquake/server/local_settings.py.tools.tmpl

```diff
@@ -0,0 +1,41 @@
+# This feature requires:
```

Comment:
Please rename this file to local_settings.py.tools (without tmpl) to make it consistent with other existing templates

Reply:
Solved with this commit: https://github.com/gem/oq-engine/pull/9531/commits/64022863b096c32cc6a1ebaf786b97bd98e9f545